### DISCUSSION TITLE
导入后未显示XCharts下拉选单

### DISCUSSION THREAD

**Discussion by jessica-peng-github** [XCharts-Team/XCharts#258] [Q&A] [answered]
我下载好XCharts3.0源码后，直接将XCharts目录拷贝到Unity项目工程的Assets目录下。但如图所示，在Hierarchy视图下右键或菜单栏GameObject下拉选择中都没有看到XCharts的选项。想请问解决的方法，谢谢?
![1](https://github.com/XCharts-Team/XCharts/assets/122681237/a65ca2af-0273-4c3d-9d69-efdfadd49392)
![2](https://github.com/XCharts-Team/XCharts/assets/122681237/16180e5c-c211-40e4-b86b-31381b01043c)

**Comment by monitor1394** [comment, marked as answer]
编译报错了，先解决代码报错。
